src/store/polls/getters.ts

```diff
@@ -7,4 +7,10 @@ export const getters: GetterTree<PollsStateInterface, RootStateInterface> & Gett
   getPolls: (state) => {
     return state.cooperationPolls;
   },
+  getSelectedPoll: (state) => {
+    return state.selectedPoll;
+  },
+  getPollByID: (state) => {
+    return state.pollById;
```

Comment:
What data is in the 'pollById' property? And where do you pass the ID?

Reply:
already fixed